### PR TITLE
update address format demostration picture.

### PR DESCRIPTION
**Pull request by CipherWang**

### CODE REVIEW

File: rfcs/0021-ckb-address-format/0021-ckb-address-format.md [hunk 1]
```diff
@@ -89,16 +89,22 @@ The human-readable part is "**ckb**" for CKB mainnet, and "**ckt**" for the test
...
-    args = ['dde7801c073dfb3464c7b1f05b806bb2bbb84e99', '00c1ddf9c135061b7635ca51e735fc2b03cee339']
-    address = "ckb1qfy29n383kzwzyptvlgp4j9z8vc6s8x9f6fzu0dnaj2d9mzr2mr8c9xau7qpcpealv6xf3a37pdcq6ajhwuyaxg5qrqam7wpx5rpka34efg7wd0u9vpuaceeu5fsh5"
+```yml
+== short address (code_hash_index = 0x00) test ==
+args to encode:          b39bbc0b3673c7d36450bc14cfcdad2d559c6c64
+address generate:        ckb1qyqt8xaupvm8837nv3gtc9x0ekkj64vud3jqfwyw5v
+
+== short address (code_hash_index = 0x01) test ==
+multi sign script:       0 | 1 | 2 | 3 | bd07d9f32bce34d27152a6a0391d324f79aab854 | 094ee28566dff02a012a66505822a2fd67d668fb | 4643c241e59e81b7876527ebff23dfb24cf16482
+script encoded:          30313233bd07d9f32bce34d27152a6a0391d324f79aab854094ee28566dff02a012a66505822a2fd67d668fb4643c241e59e81b7876527ebff23dfb24cf16482
```
Comment (yangby-cryptape) [inline rfcs/0021-ckb-address-format/0021-ckb-address-format.md]: > `S | R | M | N` are four single byte unsigned integers, ranging from 0 to 255

So, here, `0` should be `0x00`, not `0x30` (`0` in ASCII)。
[Click HERE to review implementation of the multi-signature system script.](https://github.com/nervosnetwork/ckb-system-scripts/blob/0b82741/c/secp256k1_blake160_multisig_all.c#L122)

File: rfcs/0021-ckb-address-format/0021-ckb-address-format.md [hunk 2]
```diff
@@ -89,16 +89,22 @@ The human-readable part is "**ckb**" for CKB mainnet, and "**ckt**" for the test
...
+args to encode:          38c0ae47fd1ab954d97beb873fb54ee3670b742a
+address generate:        ckb1qyqkpcpr3au0ve09wzrp5q7cn96hl8d7jwls47g2ys
```
Comment (yangby-cryptape) [inline rfcs/0021-ckb-address-format/0021-ckb-address-format.md]: My result of this generated address is `ckb1qyqkn5zvc8xvaf3g9ca5wv0f473gr82gezdqvh9tqq`.

**Comment by yangby-cryptape** [comment]
@TheWaWaR
Please do double check for all generated addresses.
We have to ensure the RFC is 100% correct.

File: rfcs/0021-ckb-address-format/0021-ckb-address-format.md (hunk 1 above)
Comment (CipherWang) [reply rfcs/0021-ckb-address-format/0021-ckb-address-format.md]: fixed

File: rfcs/0021-ckb-address-format/0021-ckb-address-format.md (hunk 2 above)
Comment (CipherWang) [reply rfcs/0021-ckb-address-format/0021-ckb-address-format.md]: fixed

**Pull request merged**